### PR TITLE
Assignment2

### PR DESCRIPTION
**Pull request by juanjoLdlF**

### CODE REVIEW

File: Assignment2/juanjoLdlF-JuanJoseLucasdelaFuente.ttl [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	ejer2:hasOwner	ejer2:User10A
+
+ejer2:User10A
+	ejer2:hasName	ejer2:"Pedro"
+	
+ejer2:Sensor029
+	ejer2:hasMeasurement	ejer2:Measurement8401
+
+ejer2:Measurement8401
+	ejer2:hasTemperature	ejer2:"29"
```
Comment (ocorcho) [inline Assignment2/juanjoLdlF-JuanJoseLucasdelaFuente.ttl]: This and the next one should be typed

File: Assignment2/juanjoLdlF-JuanJoseLucasdelaFuente.ttl [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+ejer2:User10A
+	ejer2:hasName	ejer2:"Pedro"
+	
+ejer2:Sensor029
+	ejer2:hasMeasurement	ejer2:Measurement8401
+
+ejer2:Measurement8401
+	ejer2:hasTemperature 29
+	ejer2:atTime "2010-06-12T12:00:12"^^xsd:dataTime;
```
Comment (ocorcho) [inline Assignment2/juanjoLdlF-JuanJoseLucasdelaFuente.ttl]: xsd:dateTime

File: Assignment2/juanjoLdlF-JuanJoseLucasdelaFuente.ttl [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+	ejer2:hasOwner	ejer2:User10A
+
+ejer2:User10A
+	ejer2:hasName	ejer2:"Pedro"
+	
+ejer2:Sensor029
+	ejer2:hasMeasurement	ejer2:Measurement8401
+
+ejer2:Measurement8401
+	ejer2:hasTemperature 29
```
Comment (ocorcho) [inline Assignment2/juanjoLdlF-JuanJoseLucasdelaFuente.ttl]: lacking a final .

**Comment by ocorcho** [comment]
Are you going to review the changes that I proposed?

**Comment by juanjoLdlF** [comment]
Yes, this night i'm going to make it. Sorry for the delay.

File: Assignment2/juanjoLdlF-JuanJoseLucasdelaFuente.ttl [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+ejer2:User10A
+	ejer2:hasName	ejer2:"Pedro"^^xsd:string
+	
+ejer2:Sensor029
+	ejer2:hasMeasurement	ejer2:Measurement8401
+
+ejer2:Measurement8401
+	ejer2:hasTemperature "29"^^xsd:int
+	ejer2:atTime "2010-06-12T12:00:12"^^xsd:dataTime
```
Comment (ocorcho) [inline Assignment2/juanjoLdlF-JuanJoseLucasdelaFuente.ttl]: This is dateTime. It should also finish with ".". I recommend you to check other student submissions that have been already incorporated in the repository to check these small details. Same in the previous line.

File: Assignment2/juanjoLdlF-JuanJoseLucasdelaFuente.ttl [hunk 5]
```diff
@@ -1,21 +1,21 @@
-@base <http://www.ontogrid.net/Ejercicio2#>
-@prefix ejer2: <http://www.ontogrid.net/Ejercicio2#>
-@prefix xsd: <http://www.w3.org/2001/XMLSchema#> 
+@base <http://www.ontogrid.net/Ejercicio2#> .
```
Comment (ocorcho) [inline Assignment2/juanjoLdlF-JuanJoseLucasdelaFuente.ttl]: No, this final "." is not necessary. I suggest that you use an RDF validation service before submitting again, please

**Pull request closed without merging**